frontend/lib/src/components/elements/DeckGlJsonChart/useDeckGl.tsx

```diff
@@ -96,10 +96,52 @@ const interpolate = (info: PickingInfo, body: string): string => {
   return body
 }
 
+const getShouldUseOriginalFillFunction = (
+  selectedIndices: number[],
+  anyLayersHaveSelection: boolean,
+  visualLayerSelectionMode: VisualLayerSelectionMode
+): boolean => {
+  switch (visualLayerSelectionMode) {
+    case "perLayer":
+      return selectedIndices.length === 0
+    case "perMap":
+      return !anyLayersHaveSelection
+    default:
+      assertNever(visualLayerSelectionMode)
+      return false
+  }
+}
+
 export const useDeckGl = (props: UseDeckGlProps): UseDeckGlShape => {
-  const { element, height, isLightTheme, width } = props
+  const { element, height, isLightTheme, theme, widgetMgr, width } = props
   const { tooltip, useContainerWidth: shouldUseContainerWidth } = element
 
+  //#region QueryString Parsing
+  // TODO: Remove this region. This is only here for easy testing while we iterate on our options.
```

Comment:
also, should this region still be here given this comment?

Reply:
Great call-out, see my added note above, will remove before checking-in!